docs/docs/connect/data-source/bigquery.md

```diff
@@ -26,6 +27,27 @@ Follow these steps to configure you local environemnt credentials:
 
 You have now configured Google Cloud access from your local environment. Rill will detect and use your credentials the next time you try to ingest a source.
 
+### google_application_credentials
+
+Alternatively, we recommend creating explicit credentials in plug into Rill as this will facilitate an easier deploy to Rill Cloud. Assuming you've followed step 1 and 2 above, you'll need to create your json credential with the following command.
```

Comment:
"in plug into" is broken grammar

Reply:
> you'll need to create your json credential

It's unclear what "your json credential" is referring to. What about saying a few words about what the `google_application_credentials` file actually is?